internal/commands/validate.go

```diff
@@ -19,9 +19,10 @@ type validateOptions struct {
 func validateCmd() *cobra.Command {
 	var opts validateOptions
 	cmd := &cobra.Command{
-		Use:   "validate [APP_NAME] [--set KEY=VALUE ...] [--parameters-file PARAMETERS_FILE]",
-		Short: "Checks the rendered application is syntactically correct",
-		Args:  cli.RequiresMaxArgs(1),
+		Use:     "validate [APP_DEFINITION] [--set KEY=VALUE ...] [--parameters-file PARAMETERS_FILE]",
+		Short:   "Check that a .dockerapp App Definition is syntactically correct",
```

Comment:
'a .dockerapp App Definition' or just 'an App Definition' ?

Reply:
I was wondering about that. That's true that "an App Definition" would be enough but I added ".dockerapp" to be more self-explanative... Maybe you're right I should simplify.

Reply:
Proposal: "Check that an App Definition (.dockerapp) is is syntactically correct"
Is it better?